File "bench/check-ce/func_call.mlw", line 8, characters 12-21:
Sub-goal Postcondition of goal g'vc.
Prover result is: Valid (5 steps).

<check_ce:categorization>Categorizations of models:
- Checked model 0: BAD_CE
  - Concrete RAC: NORMAL
  - Abstract RAC: STUCK (failure in postcondition of `g` at "bench/check-ce/func_call.mlw", line 8, characters 12-21)
- Selected model 1: SW
  - Concrete RAC: NORMAL
  - Abstract RAC: FAILURE (assertion at "bench/check-ce/func_call.mlw", line 14, characters 11-16)
- Checked model 2: SW
  - Concrete RAC: NORMAL
  - Abstract RAC: FAILURE (assertion at "bench/check-ce/func_call.mlw", line 14, characters 11-16)
File "bench/check-ce/func_call.mlw", line 14, characters 11-16:
Sub-goal Assertion of goal f'vc.
Prover result is: Unknown (unknown) (6 steps).
The contracts of the following functions/loops are too weak :
  - Function 'g' at "bench/check-ce/func_call.mlw", line 13, characters 2-6
, for example during the following execution:
File int.mlw:
  Line 13:
    Constant zero initialization
    zero = 0
  Line 14:
    Constant one initialization
    one = 1
File func_call.mlw:
  Line 4:
    Constant z initialization
    (giant-step) execution of unimplemented function with args:
    result = 0
    Normal execution of function `ref` with args:
      contents = 0
    Normal execution of function `ref'mk` with args:
      contents = 0
    z = {contents= 0}
Unknown location:
    _ = ()
File func_call.mlw:
  Line 11:
    _ = ()
    Execution of main function `f` with env:
      z = {contents= 0}
      _ = ()
      zero = 0
      one = 1
  Line 13:
    Normal execution of function `Tuple0` with args:
    Giant-step execution of function `g` with args:
      _ = ()
    z = {contents= 3}
    result of `g` = ()
  Line 14:
    Property failure at assertion with:
      z = {contents= 3}

